CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Connector widget sets the default alias as "default" if the `.ini` file has no connections
 * Connector widget does not modify `.ini` file if the connection fails (#68)
 * Connector widget allows editing connections (#61)
+* Added Oracle, Microsoft SQLServer, Redshift in DB templates.
```

Comment:
please create a new `## 0.2.2` section in the changelog and move your entry there (we haven't automated this changelog management here and it sometimes happen that the last section isn't the current one but the last we released)

Reply:
[fixed](https://github.com/ploomber/jupysql-plugin/pull/73/commits/d1b06bf50f3214c22a94193ec492546878f95e28)